shell/platform/android/io/flutter/plugin/editing/TextInputPlugin.java

```diff
@@ -418,6 +418,8 @@ private void setPlatformViewTextInputClient(int platformViewId) {
     mRestartInputPending = false;
   }
 
+  // Called by the text input channel to update the text input plugin with the
+  // latest TextEditState from the framework.
```

Comment:
Nit: TextEditState => TextEditingState

Reply:
The class that this method takes as a parameter is called `TextEditState`.